app/workers/publishing_api_withdrawal_worker.rb

```diff
@@ -1,11 +1,18 @@
 class PublishingApiWithdrawalWorker < PublishingApiWorker
   def perform(content_id, explanation, locale, allow_draft = false)
+    unpublished_at = Edition
```

Comment:
Is this right? This is returning the `updated_at` of the first `Edition` of the `Document` if I'm reading it correctly? If you're going to derive it like this wouldn't it be the last edition that we need? We could also get the last `Edition` in `withdrawn` state to be more certain. Or... get the latest `Unpublishing` but that would be another join.